src/human_player_module.c

```diff
@@ -104,18 +104,26 @@ ret_type_e record_human_input(void)
     }
     else if (user_input[0] == 'r' || user_input[0] == 'R')
```

Comment:
Please change "r" to other character, since "r" is also for color RED.

Reply:
Done